shacl/BrickShape.ttl

```diff
@@ -69,6 +69,16 @@ bsh:hasPointRangeShape a sh:NodeShape ;
             sh:path brick:hasPoint ] ;
     sh:targetSubjectsOf brick:hasPoint .
 
+bsh:hasSiteHasLocationShape a sh:NodeShape ;
+    sh:property [ sh:class brick:Site ;
+            sh:message "Subject of property hasSite has object with incorrect type" ;
+            sh:path brick:hasLocation ],
+        [ sh:class brick:Site ;
+            sh:message "Subject of property isSiteOf has object with incorrect type" ;
+            sh:path brick:hasLocation ] ;
```

Comment:
It looks like these are getting added twice when they should just be there once --- can you investigate? It also looks like it is targeting subjects of both hasSite and isSiteOf, which is incorrect